### DISCUSSION TITLE
可以使用微信公众自己的素材库吗

### DISCUSSION THREAD

**Discussion by honwhy** [doocs/md#303] [General]
可以使用微信公众自己的素材库吗
既然是写微信公众号文章了，不想自己再去其他地方搞个图片库了

**Comment by honwhy** [comment]
啊哈，差点忘记了这里回应下。
经过多次研究，我决定用插件形式实现了，
fork：https://github.com/honwhy/md/tree/md-extension
主页：https://mpmd.pages.dev
解释：https://mp.weixin.qq.com/s/Y12gMSHGFQLTgNihnGJukA

另外的灵感：为解决公众号图片的跨域、防盗链问题，可以使用https://images.weserv.nl/ 解决，
从这里得到的灵感：https://github.com/caspartse/eBooksAssistant/blob/2024/references/API_Reference.md
![image](https://github.com/user-attachments/assets/dc40a27b-df15-4938-bd79-ec5135bcb0b0)

**Reply by yanglbme** [reply]
感谢反馈
